bootstrap/bootstrap.sh

```diff
@@ -28,8 +29,54 @@ sudo apt-get update
 sudo apt-get -y dist-upgrade
 
 #----nodejs install
-curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
 sudo apt-get install -y nodejs
 
-#----alsa install
-sudo apt-get install -y alsa-base alsa-utils
+#----install official requirements
+sudo apt-get install -y alsa-base alsa-utils libasound2-dev
+
+#----install missing pigpio in Raspbian Lite (the command npm install pigpio will be exec by package.json)
+sudo apt-get install pigpio
+
+#----install git and download tjbot
+if [ ! -d "${TJBOT_FOLDER}/recipes/conversation" ]; then
+    sudo apt-get install -y git
+
+    rm -Rf /home/pi/git/tjbot
+    git clone https://github.com/ibmtjbot/tjbot.git /home/pi/git/tjbot
+
+    TJBOT_FOLDER='/home/pi/git/tjbot'
+fi
+
+#----install conversation (to install it will resolve tjbotlib and other dependencies)
+cd $TJBOT_FOLDER
+cd recipes/conversation
+echo "path: $PWD"
+
+sudo npm install --unsafe-perm -g node-gyp
```

Comment:
@jweisz @victordibia I tested with npm 7.x to see if it resolve the warning and deprecated messages, but not lucky.

I also see that the packages that are causing these messages are out of control of TJBot. It is some dependencies of dependencies.

So we have to decide what to do here to move on. 
I revert the last commit to use "npm install". 
It is already tested by me and @fivdi, it is not have a beautiful output but works.
To minimize the problem I also silence the output with `npm install > install.log 2>&1` (same suggested by fivdi).

What do you think is better?
1) keep `npm install` and document the annoying messages
2) use lastest commit to hide the detail of npm install and have an output like that:

> Reading package lists... Done
> 
> ## Run `apt-get install nodejs` (as root) to install Node.js v6.x and npm
> 
> Reading package lists... Done
> Building dependency tree       
> Reading state information... Done
> The following NEW packages will be installed:
>   nodejs
> 0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
> Need to get 0 B/8,850 kB of archives.
> After this operation, 45.4 MB of additional disk space will be used.
> Selecting previously unselected package nodejs.
> (Reading database ... 32391 files and directories currently installed.)
> Preparing to unpack .../nodejs_6.10.2-1nodesource1~jessie1_armhf.deb ...
> Unpacking nodejs (6.10.2-1nodesource1~jessie1) ...
> Processing triggers for man-db (2.7.0.2-5) ...
> Setting up nodejs (6.10.2-1nodesource1~jessie1) ...
> Installing new version of config file /etc/profile.d/nodejs.sh ...
> Reading package lists... Done
> Building dependency tree       
> Reading state information... Done